packages/@aws-cdk/aws-pipes-targets-alpha/lib/stepfunctions.ts

```diff
@@ -0,0 +1,83 @@
+import { IInputTransformation, IPipe, ITarget, TargetConfig } from '@aws-cdk/aws-pipes-alpha';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import * as sfn from 'aws-cdk-lib/aws-stepfunctions';
+import { StateMachine, StateMachineType } from 'aws-cdk-lib/aws-stepfunctions';
+
+/**
+ * StateMachine StepFunction target properties.
+ */
+export interface StateMachineTargetParameters {
+  /**
+   * The input transformation to apply to the message before sending it to the target.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetparameters.html#cfn-pipes-pipe-pipetargetparameters-inputtemplate
+   * @default none
+   */
+  readonly inputTransformation?: IInputTransformation;
+
+  /**
+   * Specify whether to invoke the Step Functions state machine synchronously (`REQUEST_RESPONSE`) or asynchronously (`FIRE_AND_FORGET`).
+   *
+   * @see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetsqsqueueparameters.html#cfn-pipes-pipe-pipetargetsqsqueueparameters-messagededuplicationid
+   */
+  readonly invocationType: StepFunctionInvocationType;
+}
+
+/**
+ * InvocationType for invoking StepFunction.
+ * @see https://docs.aws.amazon.com/eventbridge/latest/pipes-reference/API_PipeTargetStateMachineParameters.html
+ */
+export enum StepFunctionInvocationType {
+  /**
+   * Invoke StepFunction asynchronously (`StartExecution`). See https://docs.aws.amazon.com/step-functions/latest/apireference/API_StartExecution.html for more details.
+   */
+  FIRE_AND_FORGET = 'FIRE_AND_FORGET',
+
+  /**
+   * Invoke StepFunction synchronously (`StartSyncExecution`) and wait for the execution to complete. See https://docs.aws.amazon.com/step-functions/latest/apireference/API_StartSyncExecution.html for more details.
+   */
+  REQUEST_RESPONSE = 'REQUEST_RESPONSE',
+}
+
+/**
+ * An EventBridge Pipes target that sends messages to a StepFunction.
+ */
+export class StepFunctionTarget implements ITarget {
+  public readonly targetArn: string;
+
+  private readonly stateMachine: sfn.IStateMachine;
+  private readonly invocationType: StepFunctionInvocationType;
+  private readonly inputTemplate?: IInputTransformation;
+
+  constructor(stateMachine: sfn.IStateMachine, parameters: StateMachineTargetParameters) {
+    this.stateMachine = stateMachine;
+    this.targetArn = stateMachine.stateMachineArn;
+    this.invocationType = parameters.invocationType;
+    this.inputTemplate = parameters.inputTransformation;
+
+    if (this.stateMachine instanceof StateMachine
+      && this.stateMachine.stateMachineType === StateMachineType.STANDARD
+      && this.invocationType === StepFunctionInvocationType.REQUEST_RESPONSE) {
+      throw new Error('Standard Workflows do not support REQUEST_RESPONSE invocation type');
```

Comment:
Can you phrase this to include a solution for the user.
```suggestion
      throw new Error('STANDARD state machine workflows do not support the REQUEST_RESPONSE invocation type. Use FIRE_AND_FORGET instead.');
```

Also let's default to FIRE_AND_FORGET

Reply:
adjusted to your suggestion and defaulted to `FIRE_AND_FORGET`